Orlando Walking Tours/Controllers/AddTourVC.swift

```diff
@@ -68,14 +70,8 @@ class AddTourVC: UIViewController, UITextFieldDelegate
     @IBAction func addTourButtonTapped(sender: UIButton)
     {
         // This block with create and save a Tour entity on a background thread
-        MagicalRecord.saveWithBlock(
-        { context in
-            self.uuid = NSUUID()
-
-            let tour = Tour.MR_createEntityInContext(context)
-            tour?.title = self.tourNameTextField.text
-            tour?.uuid = self.uuid?.UUIDString
-        })
+        self.uuid = NSUUID()
+        modelService.createTour(uuid: self.uuid!, title: tourNameTextField.text!)
```

Comment:
I'm thinking about modifying the `createTour()` function a little...  i don't quite like having to create a UUID first and then passing it into the create function.  I think i might move the creating into the function itself and just put the UUID into the completion handler instead.  I feel like that would be better encapsulation.  Thoughts?